docs/reference/modules/cluster/shards_allocation.asciidoc

```diff
@@ -45,15 +45,15 @@ one of the active allocation ids in the cluster state.
      (<<dynamic-cluster-setting,Dynamic>>)
      How many concurrent outgoing shard recoveries are allowed to happen on a
      node. Outgoing recoveries are the recoveries where the source shard (most
-     likely the primary unless a shard is relocating) is allocated on the node.
+     likely the replica unless a shard is relocating) is allocated on the node.
```

Comment:
`primary` seems correct to me - the primary is always the source shard for peer recoveries.

Reply:
Err, you are right. My understanding was off.